packages/http-client-js/test/e2e/http/type/property/additional-properties/spreads.test.ts

```diff
@@ -377,7 +377,10 @@ describe("Missing AdditionalProperties Endpoints", () => {
     });
   });
 
-  describe("SpreadRecordDiscriminatedUnion", () => {
+  /**
+   * Deprecated @discriminator in unions
+   */
+  describe.skip("SpreadRecordDiscriminatedUnion", () => {
```

Comment:
is that just to migrate later?

Reply:
Yeah once http-specs is updated I'll update this